src/main/java/bottle/Ui.java

```diff
@@ -0,0 +1,112 @@
+package bottle;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import bottle.task.Task;
+import bottle.task.TaskList;
+
+/**
+ * The type Ui.
+ */
+public class Ui {
+    /**
+     * The constant lineBreak.
+     */
+    final static String lineBreak = "\n____________________________________________________________\n";
+    /**
+     * The Welcome msg.
+     */
+    private final String welcomeMsg =
+            " Hello! I'm bottle.Bottle\n" +
+                    " What can I do for you?";
```

Comment:
```suggestion
            " Hello! I'm bottle.Bottle\n" 
             + " What can I do for you?";
```

cause if not checkstyle screams at you :}